pkg/types/kubevirt/defaults/platform.go

```diff
@@ -0,0 +1,11 @@
+package defaults
+
+import (
+	"github.com/openshift/installer/pkg/types/kubevirt"
+)
+
+// SetPlatformDefaults sets the defaults for the platform.
+func SetPlatformDefaults(p *kubevirt.Platform) {
+	// No default values to set - all values are mandatory or empty value is the default
+	// For example StorageClass and PersistentVolumeAccessMode are optional, but empty string is valid default value
```

Comment:
The purpose of this function is to **replace** the empty string with the default value.

Reply:
Yes, I understand that
But for both StorageClass and PersistentVolumeAccessMode empty string is the default value
See the following comments for more info:
https://github.com/openshift/installer/pull/4350#discussion_r528295188
https://github.com/openshift/installer/pull/4350#discussion_r528310078

Reply:
I changed it to set PersistentVolumeAccessMode default here